File: AELP/ClientApp/src/app/Models/Structure.ts
import { Node } from './Node';
import { Element } from './Element';
import { NodalLoad } from './NodalLoad';
import { ElementLoad } from './ElementLoad';

export class Structure {
    constructor(
        public nodes: Node[],
        public elements: Element[],
        public nodalLoads: NodalLoad[],
        public elementLoads: ElementLoad[]
    ) {}
}

File: AELP/Models/Structure.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AELEP.Models
{
    /// <summary>
    /// Representa a estrutura a ser analisada.
    /// </summary>
    public class Structure
    {
        [JsonProperty("nodes")]
        public List<Node> Nodes { get; set; }

        [JsonProperty("elements")]
        public List<Element> Elements { get; set; }

        [JsonProperty("nodalLoads")]
        public List<NodalLoad> NodalLoads { get; set; }

        [JsonProperty("elementLoads")]
        public List<ElementLoad> ElementLoads { get; set; }
    }
}
